### ISSUE TITLE
aws ecr get-login deprecated 

### ISSUE THREAD

**Issue by rafilkmp3** [jwalton/gh-ecr-push#10]
Hi
`aws ecr get-login `
This command is deprecated. Use get-login-password instead

reference 

https://docs.aws.amazon.com/cli/latest/reference/ecr/get-login.html

**Comment by rafilkmp3** [comment]
Today old command was removed 

```
$ $(aws ecr get-login --no-include-email --region us-east-2)

usage: aws [options] <command> <subcommand> [<subcommand> ...] [parameters]
To see help text, you can run:

  aws help
  aws <command> help
  aws <command> <subcommand> help

aws: error: argument operation: Invalid choice, valid choices are:

batch-check-layer-availability           | batch-delete-image                      
batch-get-image                          | complete-layer-upload                   
create-repository                        | delete-lifecycle-policy                 
delete-repository                        | delete-repository-policy                
describe-image-scan-findings             | describe-images                         
describe-repositories                    | get-authorization-token                 
get-download-url-for-layer               | get-lifecycle-policy                    
get-lifecycle-policy-preview             | get-repository-policy                   
initiate-layer-upload                    | list-images                             
list-tags-for-resource                   | put-image                               
put-image-scanning-configuration         | put-image-tag-mutability                
put-lifecycle-policy                     | set-repository-policy                   
start-image-scan                         | start-lifecycle-policy-preview          
tag-resource                             | untag-resource                          
upload-layer-part                        | get-login-password                      
wait                                     | help                                    

child_process.js:660
    throw err;
    ^
```

**Comment by rafilkmp3** [comment]
I think this fixes too
https://github.com/jwalton/gh-ecr-push/pull/9

**Comment by jwalton** [comment]
Fixed by #11, published in v1.3.0.
